### PR TITLE
CORE-3283 output error message prior to abort install

### PR DESCRIPTION
**Pull request by stereomon**
Added outputBuffer which is used to flush messages when an exception occurs. 

BC branch: https://github.com/spryker/shop-suite/pull/31

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -0,0 +1,3836 @@
+{
+    "_readme": [
```
Comment (dereuromark) [inline composer.lock]: This file should be excluded (is so in gitignore already afaik)

Comment (stereomon) [reply composer.lock]: I would not ignore the composer.lock file, especially if there is no problem with it.

Comment (dereuromark) [reply composer.lock]: There is. also for ci testing. Please remove it.

Comment (stereomon) [reply composer.lock]: Then please explain what the current problem is.

Comment (dereuromark) [reply composer.lock]: We discussed that many times before for non project (libraries) packages.
Here also composer install will not properly pull latest (which it must!)

**Pull request merged**